File: app/src/main/java/com/faendir/lightning_launcher/multitool/animation/PointB.java
package com.faendir.lightning_launcher.multitool.animation;

/**
 * @author lukas
 * @since 10.07.18
 */
class PointB {
    boolean x;
    boolean y;

    PointB(boolean x, boolean y) {
        this.x = x;
        this.y = y;
    }

    boolean any() {
        return x || y;
    }

    boolean both() {
        return x && y;
    }
}

File: app/src/main/java/com/faendir/lightning_launcher/multitool/animation/PointB.kt
package com.faendir.lightning_launcher.multitool.animation

/**
 * @author lukas
 * @since 10.07.18
 */
class PointB(val x: Boolean, val y: Boolean) {

    fun any(): Boolean = x || y

    fun both(): Boolean = x && y
}
